### PR TITLE
adding filtering and compute documentation

### PR DESCRIPTION
**Pull request by felipeZ**
##  New
Add documentation for filtering and computing molecular properties for potential quantum dots ligands.

## Remove
Unused files and badges from the README

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,40 +0,0 @@
-name: Build
```
Comment (BvB93) [inline .github/workflows/build.yml:1]: We probably want to reintroduce this file at some point in the future,
once we add a test for generating the docs.
Removing it should be good for now though.

Comment (felipeZ) [reply .github/workflows/build.yml:1]: Agree

File: setup.py [hunk 2]
```diff
@@ -35,14 +34,13 @@
         'License :: OSI Approved :: Apache Software License',
         'Natural Language :: English',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.6',
```
Comment (BvB93) [inline setup.py:38]: Btw, is there a specific reason for dropping 3.6?

Comment (felipeZ) [reply setup.py:38]: Very few people (that I am aware of) keep on using "old" python versions, most people use either the current stable version (3.8 in this moment) or the previous one (3.7). But apart from that I don't have any other reason

Comment (BvB93) [reply setup.py:38]: I suppose that's fair.
I would recommend keeping `python_requires='>=3.6'` though, at least until we encounter a concrete situation where 3.6 is inadequate (at which point we can fully drop it without a second thought).

**Comment by felipeZ** [comment]
@juliette1996  and @BvB93, I am done with the PR. Let me know if you agree with the changes :)

**Review by BvB93 — APPROVED** [review]
Looks good to me, thanks Felipe!

**Pull request merged**